### PR TITLE
Update Windows Documentation

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
Quick update in our windows documentantion

##### Test Plan

Check and propose grammar fixes.

##### Additional Information

<details> <summary>For users: How does this change affect me?</summary>
  
</details>

### CODE REVIEW

File: packaging/windows/WINDOWS_INSTALLER.md [hunk 1]
```diff
@@ -6,55 +6,35 @@ Netdata offers a convenient Windows installer for easy setup. This executable pr
...
 
-This section provides instructions for installing Netdata in silent mode, which is ideal for automated deployments.
+This section provides instructions for installing Netdata in silent mode running `msiexec.exe`, which is ideal for automated deployments.
 
 > **Info**
 >
 > Run the installer as admin to avoid the Windows prompt.
 >
-> Silent mode skips displaying license agreements, but requires explicitly accepting them using the `/A` option.
+> Silent mode skips displaying license agreements, but requires explicitly accepting them using the arguments `GPLLICENSE=1` and `CLOUDLICENSE=1`.
```
Comment (ilyam8) [inline packaging/windows/WINDOWS_INSTALLER.md:17]: There is no Cloud license. I asked to rename it multiple times. @stelfrag can you help us please?

Comment (thiagoftsm) [reply packaging/windows/WINDOWS_INSTALLER.md]: Renamed.

**Comment by Ancairon** [comment]
We need to update also the updating doc, and I think the uninstalling, will check tomorrow and approve

**Comment by thiagoftsm** [comment]
Perfect @Ancairon ,

We also plan to merge https://github.com/netdata/netdata/pull/18903 before it.

**Review by Ferroin — APPROVED** [review]

**Pull request merged**